src/scripts/achievements/OakItemRequirement.ts

```diff
@@ -0,0 +1,15 @@
+///<reference path="Requirement.ts"/>
+
+class OakItemRequirement extends Requirement {
```

Comment:
Can we make this more descriptive of what's being tracked, maybe something along the lines of:
```suggestion
class MaxLevelOakItemRequirement extends Requirement {
```